react-main/src/app/screens/Game/index.js

```diff
@@ -1,17 +1,64 @@
 import React, { Component } from 'react';
 
 import Board from './components/Board';
+import { calculateWinner } from './utils';
 
 class Game extends Component {
+  state = {
+    history: [{ squares: Array(9).fill(null) }],
+    stepNumber: 0,
+    xIsNext: true
+  };
+
+  handleClick = i => {
+    const { history, xIsNext, stepNumber } = this.state;
+    const current = history.slice(0, stepNumber + 1);
+    const modifiedHistory = current[current.length - 1];
```

Comment:
This wouldn't be necessary if you kept only the wanted item in the previous line